platforms/cuda/src/CudaKernels.cpp

```diff
@@ -902,8 +907,9 @@ double CudaCalcNonbondedForceKernel::execute(ContextImpl& context, bool includeF
                     cu.getInvPeriodicBoxSizePointer(), cu.getPeriodicBoxVecXPointer(), cu.getPeriodicBoxVecYPointer(), cu.getPeriodicBoxVecZPointer(),
                     recipBoxVectorPointer[0], recipBoxVectorPointer[1], recipBoxVectorPointer[2], &pmeAtomGridIndex.getDevicePointer(),
                     &charges.getDevicePointer()};
-            cu.executeKernel(pmeSpreadChargeKernel, spreadArgs, cu.getNumAtoms(), 128);
+            cu.executeKernel(pmeSpreadChargeKernel, spreadArgs, pmeNumThreadBlocks * pmeThreadBlockSize, pmeThreadBlockSize);
```

Comment:
I think a better value for the third argument is `6*cu.getNumAtoms()`.  The goal of that argument is to set a limit on the number of threads, so it doesn't create lots of threads that have nothing to do for very small systems.  The new version of the kernel creates more threads (32 for every 6 atoms, rather than one per atom), so the limit should be higher, but it should still be limited for sufficiently small systems.